SeafarersAPI/Data/Repositories/SubmissionRepository.cs

```diff
@@ -3,56 +3,57 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
 
 namespace SeafarersAPI.Data.Repositories
 {
-    public class SubmissionRepository
+    public class SubmissionRepository : IRepository<Submission>
     {
         public SubmissionContext context;
         public SubmissionRepository()
         {
-            if(context== null)
+            if(context == null)
             {
                 context = new SubmissionContext();
             }
         }
 
-        public List<Submission> GetSubmissions()
+        public IEnumerable<Submission> GetList(int startIndex, int amountToGet)
```

Comment:
Prefer IEnumerable over list at first since the. See this StackOverflow post as to why IEnumerable over IList: https://stackoverflow.com/questions/3628425/ienumerable-vs-list-what-to-use-how-do-they-work
IEnumerable describes behavior, while List is an implementation of that behavior. When you use IEnumerable, you give the compiler a chance to defer work until later, possibly optimizing along the way. If you use ToList() you force the compiler to reify the results right away.